code/game/machinery/droneDispenser.dm

```diff
@@ -135,14 +135,9 @@
 
 /obj/machinery/droneDispenser/power_change()
 	..()
-	if(powered())
```

Comment:
I assume this is handled elsewhere?

Reply:
Yes the exact thing (`powered()` defaults to act like `powered(power_channel)`) exists at
https://github.com/BeeStation/BeeStation-Hornet/blob/1d6bacb2eefe4e7ba97b7ca74d0d39dcbfc50cd8/code/modules/power/power.dm#L107-L115
which is called at untouched `..()` above